CMakeLists.txt

```diff
@@ -266,10 +266,10 @@ endfunction(copy_module_host_dll)
 
 function(copy_iothub_client_dll whatIsBuilding whatIsBuildingLocation)
   if(WIN32)
-    if(EXISTS "${azure_iot_sdks_DIR}/../bin/iothub_client.dll")
+    if(EXISTS "${azure_iot_sdks_DIR}/../bin/iothub_client_dll.dll")
```

Comment:
I thought they changed the target to `iothub_client_dll`, but still kept the binary as `iothub_client.dll`?

Reply:
They couldn't. Creating a Windows DLL actually creates 2 files, <name>.dll and <name>.lib (3 if you count the PDB), which would conflict with the static library <name>.lib. To avoid the filename conflict, the C SDK doesn't rename the shared library.  It does on Linux, which is perfectly happy with its independent .so and .a files.

Reply:
Ah, right. I remember now that we talked about this.